modules/ROOT/pages/dataweave-share-reuse.adoc

```diff
@@ -0,0 +1,18 @@
+= Sharing and Reusing Custom DataWeave Modules and Mappings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+DataWeave enables you to implement, test, and package your custom DataWeave modules and mappings into libraries that you can share and reuse through Anypoint Exchange.
+
+xref:dataweave-extension-plugin.adoc[DataWeave 2.0 extension to Visual Studio Code] enhances productivity when you are building DataWeave libraries. The extension provides the following capabilities:
+
+* Ability to work on your DataWeave modules and mappings in a standalone manner, outside of a Mule application.
+* Instant execution of your DataWeave mappings, including live preview of data transformations.
+* Language support, including autocompletion, navigation, refactoring, formatting, quick fixes, syntax highlighting, and more.
```

Comment:
GLOBALLY avoid ending lists with "and more" and similar vague phrases. 
Because "including" implies that there is "more," adding "and more" to the sentence adds no value and possibly raises the question "What more?" in your reader's mind.
```suggestion
* Language support, including autocompletion, navigation, refactoring, formatting, quick fixes, and syntax highlighting.
```

Reply:
Remove periods at the ends of the following list items, none of which is a complete sentence: